src/components/shared/CallNodeContextMenu.js

```diff
@@ -504,6 +599,13 @@ class CallNodeContextMenuImpl extends React.PureComponent<Props> {
     const fileName =
       filePath &&
       parseFileNameFromSymbolication(filePath).path.match(/[^\\/]+$/)?.[0];
+    const showOpenDebuggerItem =
```

Comment:
should we check for the tabId here as well? I think we should (and an additional test should check it).

Reply:
I think this happens when debugging Firefox frontend code for example.

Reply:
Yeah, that's right. Updated here to check the tabId as well.